scripts/run_app.sh

```diff
@@ -24,7 +24,7 @@ fi
 
 # Output the current git revision
 if [ -z "$EQ_GIT_REF" ]; then
```

Comment:
do we still need any of this? it was used to determine if the app had changed but no is only displayed on the status page?